recipes-core/everest-config/everest-config-sil/everest.service

```diff
@@ -13,7 +13,7 @@ User=root
 LimitCORE=infinity
 WorkingDirectory=/usr/bin
 ExecStartPre=sleep 5
-ExecStart=/usr/bin/manager --log_conf /etc/everest/logging.ini --conf /etc/everest/config-sil.json --main_dir=/usr
+ExecStart=/usr/bin/manager --conf /etc/everest/config-sil.yaml --prefix=/usr
```

Comment:
not that it really matters, but the `--conf` flag also has a short form, `--conf config-sil` should be also possible